utils.py

```diff
@@ -392,6 +394,10 @@ def post_fault_record(message: dict, record_post_url: str):
     }
     headers = {"Content-type": "application/json", "Accept": "text/plain"}
     response = requests.post(url=record_post_url, data=json.dumps(payload), headers=headers)
+    if response.status_code == 200:
+        logger.info(f"Request ended with status {response.status_code}. Fault Record #{response.json().get('fault_id')} has been successfully created.")
+    else:
+        logger.error(f"Something went wrong. Fault Record from {message['url']} was not created.")
```

Comment:
This is good for now, at some point we might want to store the message URL in a csv so that we can try to reingest in a different runtime